config/filter.d/postfix-address.conf

```diff
@@ -0,0 +1,22 @@
+# Fail2Ban filter against Postfix SMTP mail address brute-forcing by spammers
+#
+#
+
+[INCLUDES]
+
+# Read common prefixes. If any customizations available -- read them from
+# common.local
+before = common.conf
+
+[Definition]
+
+_daemon = postfix/(submission/)?smtp(d|s)
+
+failregex = ^%(__prefix_line)sNOQUEUE: reject: RCPT from \S+\[<HOST>\]: 550 5\.1\.1 <\S+>: Recipient address rejected: User unknown in virtual mailbox table;.*$
```

Comment:
Regular expression `.*$` makes no sense at all.
I had said - remove it, but the top boss here says - please supply precise anchored expression...
For the discussion about anchors see https://github.com/fail2ban/fail2ban/pull/1479#issuecomment-231331273